src/prefabs/pageWithRegisterForm.js

```diff
@@ -0,0 +1,8601 @@
+                        },
+                        {
+                          value: [],
+                          label: 'Helper text',
+                          key: 'helperText',
+                          type: 'VARIABLE',
+                        },
+                        {
+                          label: 'Variant',
+                          key: 'variant',
+                          value: 'outlined',
+                          type: 'CUSTOM',
+                          configuration: {
+                            as: 'BUTTONGROUP',
+                            dataType: 'string',
+                            allowedInput: [
+                              {
+                                name: 'Standard',
+                                value: 'standard',
+                              },
+                              {
+                                name: 'Outlined',
+                                value: 'outlined',
+                              },
+                              { name: 'Filled', value: 'filled' },
+                            ],
+                          },
+                        },
+                        {
+                          type: 'TOGGLE',
+                          label: 'Full width',
+                          key: 'fullWidth',
+                          value: true,
+                        },
+                        {
+                          label: 'Size',
+                          key: 'size',
+                          value: 'Medium',
+                          type: 'CUSTOM',
+                          configuration: {
+                            as: 'BUTTONGROUP',
+                            dataType: 'string',
+                            allowedInput: [
+                              { name: 'Medium', value: 'medium' },
+                              { name: 'Small', value: 'small' },
+                            ],
+                          },
+                        },
+                        {
+                          label: 'Margin',
+                          key: 'margin',
+                          value: 'normal',
+                          type: 'CUSTOM',
+                          configuration: {
+                            as: 'BUTTONGROUP',
+                            dataType: 'string',
+                            allowedInput: [
+                              { name: 'None', value: 'none' },
+                              { name: 'Dense', value: 'dense' },
+                              { name: 'Normal', value: 'normal' },
+                            ],
+                          },
+                        },
+                        {
+                          label: 'Adornment',
+                          key: 'adornmentIcon',
+                          value: 'Email',
+                          type: 'CUSTOM',
+                          configuration: {
+                            as: 'DROPDOWN',
```

Comment:
I expected the `iconConfiguration` to be be used here

Reply:
Added `iconConfiguration`